### ISSUE TITLE
IG++ cannot disable hold to zoom feature

### ISSUE THREAD

**Issue by tonynca** [eni9889/IG-PP-Issues#384]
I cannot disable the hold to zoom feature. Even with the feature toggled off. The feature is still active within tile view. 

Hold to zoom accidentally launches when im just scrolling through my feed. Very annoying

**Comment by badger200** [comment]
Yes this is annoying. I'll be scrolling somebody's feed and suddenly it goes to a full screen black view of some photo and I can't even quickly swipe to get rid of it. I have to sit and precisely locate the X to exit.
